pkg/cmd/kafka/create/create.go

```diff
@@ -501,29 +503,64 @@ func selectEnterpriseOrRHInfraPrompt(opts *options) error {
 	if err != nil {
 		return err
 	}
-	opts.useEnterpriseCluster = idx != 0
+	opts.useEnterpriseFlow = idx != 0
+	opts.useLegacyFlow = idx == 0
 
 	return nil
 }
 
+func checkForLegacyQuota(opts *options, orgQuotas *accountmgmtutil.OrgQuotas) {
+	// Check if the user has enterprise quota
+	for _, quota := range orgQuotas.EnterpriseQuotas {
+		if quota.Quota > 0 {
+			opts.useEnterpriseFlow = true
+		}
+	}
+	// to-do may have to deal with trial (devoloper instances) quota here
+
+	// Check if the user has a legacy quota
+	for _, quota := range orgQuotas.StandardQuotas {
+		if quota.Quota > 0 {
+			opts.hasLegacyQuota = true
+			return
+		}
+	}
+	for _, quota := range orgQuotas.MarketplaceQuotas {
+		if quota.Quota > 0 {
+			opts.hasLegacyQuota = true
+			return
+		}
+	}
+	for _, quota := range orgQuotas.EvalQuotas {
+		if quota.Quota > 0 {
+			opts.hasLegacyQuota = true
+			return
+		}
+	}
+}
+
 // Show a prompt to allow the user to interactively insert the data for their Kafka
 // nolint:funlen
 func promptKafkaPayload(opts *options, constants *remote.DynamicServiceConstants) (*kafkamgmtclient.KafkaRequestPayload, error) {
 	f := opts.f
 
-	validator := &kafkacmdutil.Validator{
-		Localizer:  f.Localizer,
-		Connection: f.Connection,
+	// getting org quotas
+	orgQuota, err := accountmgmtutil.GetOrgQuotas(f, &constants.Kafka.Ams)
+	if err != nil {
+		return nil, err
 	}
 
-	promptName := &survey.Input{
-		Message: f.Localizer.MustLocalize("kafka.create.input.name.message"),
-		Help:    f.Localizer.MustLocalize("kafka.create.input.name.help"),
-	}
+	// check if org has legacy (non-hybrid) quota
+	checkForLegacyQuota(opts, orgQuota)
+
+	var enterpriseQuota accountmgmtutil.QuotaSpec
 
 	answers := &promptAnswers{}
 
-	err := survey.AskOne(promptName, &answers.Name, survey.WithValidator(validator.ValidateName), survey.WithValidator(validator.ValidateNameIsAvailable))
+	if !opts.useEnterpriseFlow {
+		f.Logger.Info(opts.f.Localizer.MustLocalize("kafka.create.info.enterpriseQuota"))
```

Comment:
So are logging to show how to get enterprise quota every time they don't use the new standard flow even if they have legacy?

Reply:
yes, seeing as enterprise is rhosak now, we want to sell it. if they don't have enterprise quota, tell them where to get it to give us money. This may change when know what the craic is with trials but until then I think this is the best method